tests/gui/test_dataviews.py

```diff
@@ -84,3 +85,45 @@ def test_table_sort_string(qtbot):
     # Make sure we can sort with both numbers and strings (i.e., "")
     table.model().sort(0)
     table.model().sort(1)
+
+def test_camera_table(qtbot, multiview_min_session_labels):
+    table = CamerasTableModel(items=multiview_min_session_labels.sessions[0])
+    
+    assert table.columnCount() == 2
+    assert table.rowCount() == 8
+
+    num_rows = table.rowCount()
+
+    table = GenericTableView(
+        row_name="instance",
+        is_sortable=True,
+        name_prefix="",
+        model=CamerasTableModel(items=multiview_min_session_labels.sessions[0]),
+    )
+
+    # Testing if all comcorders are presented in the correct row
+    camcorders = multiview_min_session_labels.sessions[0].linked_cameras
+
+    for i in range(num_rows):
+        table.selectRow(i)
+        assert table.getSelectedRowItem() == camcorders[i]
+        assert table.model().data(table.currentIndex()) == camcorders[i].name
+
+    # Testing if a comcorder change is reflected
+    idxs_to_remove = [1, 2, 7]
+    for idx in idxs_to_remove:
+        multiview_min_session_labels.sessions[0].remove_video(camcorders[idx].get_video(multiview_min_session_labels.sessions[0]))
+
+    removed_camcorder = [cam for i, cam in enumerate(camcorders) if i not in idxs_to_remove]
+
+    for i in range(num_rows-3):
+        table.selectRow(i)
+        assert table.getSelectedRowItem() == camcorders[i]
+        assert table.model().data(table.currentIndex()) == removed_camcorder[i].name
+    
+
+
+
+
+
+    
```

Comment:
Lint with `black test/gui/test_dataview/py` or set-up black to formatOnSave in vscode